crypto/fipsmodule/bn/bytes.c

```diff
@@ -210,9 +210,29 @@ void bn_assert_fits_in_bytes(const BIGNUM *bn, size_t num) {
   size_t tot_bytes = bn->width * sizeof(BN_ULONG);
```

Comment:
Can we add a comment before this function to explain its difference from `fits_in_bytes` that's right before it?

Reply:
Comment added in latest commit.